zoo-css-challenge/style.css

```
@@ -1,195 +1,248 @@
-  cursor: pointer;
-  padding: 10px 20px;
-  box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2);
-  background-color: #372772;
-  border: 2px solid black;
-  color: #fff;
+    display: inline-block;
+    border-radius: 3px;
+    cursor: pointer;
+    padding: 10px 20px;
+    box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2);
+    background-color: #372772;
+    border: 2px solid black;
+    color: #fff;
 }
 
 footer {
-  background-color: #211a1d;
-  color: #ffffff;
-  width: 100%;
-  padding: 20px;
-  display: flex;
-  justify-content: space-between;
+    background-color: #211a1d;
+    color: #ffffff;
+    width: 100%;
+    padding: 20px;
+    display: flex;
+    justify-content: space-between;
 }
 
 .introduction {
+    background-color: #fff
 }
 
 #badge {
-  height: 120px;
-  width: 120px;
-  background-color: #bbb;
-  border-radius: 50%;
-  display: inline-block;
-  position: fixed;
-  bottom: 20px;
-  right: 20px;
-
-  display: flex;
-  justify-content: center;
-  align-content: center;
-  flex-direction: column;
+    height: 120px;
+    width: 120px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+    position: fixed;
+    bottom: 20px;
+    right: 20px;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    flex-direction: column;
 }
 
 #badge:hover {
-  background-color: yellow;
+    background-color: yellow;
 }
 
 #badge p {
-  text-align: center;
-  font-size: 125%;
+    text-align: center;
+    font-size: 125%;
 }
 
 .image-container img {
-  vertical-align: top;
-  border: 5px solid #747c92;
+    vertical-align: top;
+    border: 5px solid #747c92;
 }
 
 .image-container img:first-of-type {
-  vertical-align: bottom;
+    vertical-align: top;
+    border-color: #56e39f;
+}
+
+.image-container img:last-of-type {
+    border-color: #ff3f00
 }
 
 .tigers {
-  background-color: #0b3c49;
-  color: #f0e100;
+    background-color: #45490bef;
+    color: #f0e100;
 }
 
 .giraffe {
-  background-color: #465775;
-  color: #56e39f;
+    background-color: #483C46;
+    color: #56e39f;
 }
 
 .giraffe h2 {
-  color: #ef6f6c;
+    color: #F4743B;
+}
+
+.giraffe p {
+    color: #BEEE62;
+}
+
+.giraffe p:first-child {
+    font-size: 20px;
+}
+
+.news h2 {
+    text-align: center;
 }
 
 .card {
-  display: inline-block;
-  background-color: white;
-  padding: 20px;
-  margin: 10px;
-  width: 30%;
-  border: 1px solid black;
-  border-radius: 10px;
+    display: inline-block;
+    background-color: white;
+    padding: 20px;
+    margin: 10px;
+    width: 30%;
+    border: 1px solid black;
+    border-radius: 10px;
 }
 
 .card h4 {
-  margin-bottom: 1em;
+    margin-bottom: 1em;
 }
 
 .card a {
-  text-decoration: underline;
-  font-weight: bold;
+    text-decoration: underline;
+    font-weight: bold;
 }
 
 .card time {
-  opacity: 0.5;
-  display: inline-block;
-  margin-bottom: 10px;
+    opacity: 0.5;
+    display: inline-block;
+    margin-bottom: 10px;
 }
 
+
 /*Education*/
 
+button:hover {
```

```suggestion
button:focus,
button:hover {
```
We should also add `:focus` so that when a user uses the `Tab` key to move to a button, it also changes to green